package.json

```diff
@@ -8,7 +8,9 @@
   },
   "scripts": {
     "comment": "Rather crude approach to move ts definitions into the public folder so the browser can grab them later in CascadeMain.js",
-    "move-ts-defs": "cp ./node_modules/opencascade.js/dist/opencascade.d.ts ./web/public && cp ./node_modules/three/src/Three.d.ts ./web/public && cp ./web/src/cascade/js/StandardLibraryIntellisense.ts ./web/public"
+    "move-ts-defs": "cp ./web/src/cascade/js/StandardLibraryIntellisense.ts ./web/public",
+    "move-cad-worker": "rm -r web/public/CADWorker || true && mkdir web/public/CADWorker && mkdir web/public/CADWorker/node_modules && cp -r ./web/src/cascade/js/CADWorker ./web/public && cp -r ./web/src/cascade/node_modules ./web/public/CADWorker && cp -r ./web/src/cascade/fonts ./web/public/CADWorker",
```

Comment:
This is just copying a bunch of js worker files from the git submodule and putting them in the public folder (so they can be accessed from the browser),
which reminds me I need to update the read me to add a `yarn move-cad-worker` step to the instructions.